transcoder/__init__.py

```diff
@@ -17,26 +17,240 @@
 # limitations under the License.
 #
 
+# pylint: disable=broad-except
+
+import importlib
+import logging
+import os
+import signal
+import sys
+
+from datetime import datetime
 from enum import Enum
 
-from .version import __version__
 
+from transcoder.message.MessageUtil import get_message_parser, parse_handler_config
+from transcoder.message import DatacastParser
+from transcoder.message.factory import all_supported_factory_types
+from transcoder.message.ErrorWriter import ErrorWriter, TranscodeStep
+from transcoder.output import all_output_identifiers, get_output_manager, OutputManager
+from transcoder.source import all_source_identifiers, get_message_source, Source
 
 # pylint: disable=invalid-name
+class Transcoder: # pylint: disable=too-many-instance-attributes
```

Comment:
I suggest making Transcoder its own file and import it to keep the __init__ file to init focused code

Reply:
Great suggestion, will move to separate file.